doc/configuration.rst

```diff
@@ -1147,13 +1147,20 @@ Arguments:
 Used by:
   - `DockerDriver`_
 
+.. _udev-matching:
+
 udev Matching
 ~~~~~~~~~~~~~
-udev matching allows labgrid to identify resources via their udev properties.
+
+labgrid allows the exporter (or the client-side environment) to match resources
+via udev rules.
 Any udev property key and value can be used, path matching USB devices is
 allowed as well.
-This allows exporting a specific USB hub port or the correct identification of
-a USB serial converter across computers.
+
+The udev resources become available as soon as they are plugged into the
+computer, e.g. allowing an exporter to export all USB ports on a specific hub
```

Comment:
We shouldn't say that all USB ports are exported, as that could be confusing (because the resources need to be defined instead).